samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -952,12 +1004,12 @@ function saveBatchedTileset(tilesetName, tileOptions, tilesetOptions) {
                 var dataUri = new DataUri();
                 dataUri.format('.b3dm', b3dm);
```

Comment:
`BatchedWithContentDataUri` is not getting written as a data uri in non-legacy mode